pkg/snapstore/snapshot.go

```diff
@@ -106,6 +112,21 @@ func ParseSnapshot(snapPath string) (*Snapshot, error) {
 func (s SnapList) Len() int      { return len(s) }
 func (s SnapList) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s SnapList) Less(i, j int) bool {
+	// Ignoring errors here, as we assume at this stage the error won't happen.
+	iCreationTime, err := s[i].GetSnapshotDirectoryCreationTimeInUnix()
+	if err != nil {
+		logrus.Errorf("Failed to get snapshot directory creation time for snapshot: %s, with error: %v", path.Join(s[i].SnapDir, s[i].SnapName), err)
+	}
+	jCreationTime, _ := s[j].GetSnapshotDirectoryCreationTimeInUnix()
```

Comment:
Why aren't we using the error here? The `err` on the next line will end up referring to the previous error from `iCreationTime` right?

Reply:
done.